src/bundle/Resources/public/scss/_forms.scss

```diff
@@ -17,16 +17,15 @@ form:not(.form-inline) {
 
     .col-form-legend {
         font-weight: 700;
-        font-size: 14px;
-        color: $ez-color-base-dark;
+        font-size: calculateRem(16px);
+        color: $ibexa-black;
         margin-bottom: 0;
-        margin-top: 20px;
+        margin-top: calculateRewm(20px);
 
         @include label-required();
```

Comment:
Also be careful, mixins should be at the start, as at the moment mixin is overwriting your properties